package.json

```diff
@@ -6,15 +6,15 @@
   },
   "scripts": {
     "test": "jest",
-    "test:full": "npm run build && npm run test:coverage",
+    "test:full": "npm run test:coverage",
```

Comment:
Notice that I removed the build command here. Performing tests do not require the project to build IIRC. This can save us about 2 minutes from building the project. The script will eventually `yarn build` in the "build" folder, which creates a "dist" folder in the project directory. The contents of this folder is what we eventually deploy.